tests/asserts/util.py

```diff
@@ -33,6 +33,12 @@ def get_directory_name_from_publ_type(publ_type):
     return publ_type.split('.')[1] + 's'
 
 
+def get_publication_by_uuid(publication):
```

Comment:
```suggestion
def get_publication_resp_by_publication(publication):
```

Reply:
But we probably should not create any such method and rather store `uuid` in the test.